.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+# Notes:
+#  - All software installed below is pinned so they don't get upgraded and break w/o us knowing why.
+#  - If you bump one version up to latest, please bump them all so we stay current!
+
+name: Build medic-couch2pg and run unit test cases
+
+on: [push, pull_request]
+
+jobs:
```

Comment:
The old travis config had a notification step, but I don't see that here. Is that an intentional omission?

Reply:
This is configured in the repo [settings](https://docs.github.com/en/github/managing-subscriptions-and-notifications-on-github/configuring-notifications#github-actions-notification-options), I'm already getting the emails when it fails :)